### DISCUSSION TITLE
Standard Notes Import

### DISCUSSION THREAD

**Discussion by ejjett** [streetwriters/notesnook#3215] [Q&A] [answered]
When I import my standard notes data, none of the tags import with the notes. Is this a bug, something standard notes isn't allowing to export, or something I may be doing wrong?

Thank you for the help.

**Comment by thecodrr** [comment]
SN's backups are a mess with a lot of information all over the place. Tags do get imported but I am pretty sure our Importer is missing a few edge cases. The problem, though, is that i don't have a sample of a SN backup to reproduce and fix this issue.

**Comment by ejjett** [comment]
If there's any way I can help, let me know. Is it as simple as creating a bunch of notes with tags in standard notes and exporting the file for you to play with?

On Sat, Aug 26, 2023, at 05:54, Abdullah Atta wrote:
> 
> 
> SN's backups are a mess with a lot of information all over the place. Tags do get imported but I am pretty sure our Importer is missing a few edge cases. The problem, though, is that i don't have a sample of a SN backup to reproduce and fix this issue.
> 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/streetwriters/notesnook/discussions/3215#discussioncomment-6829974>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BCEXSLLCJ2RHUXK2WEJLCGDXXHWYHANCNFSM6AAAAAA37A7E6I>.
> You are receiving this because you authored the thread.Message ID: ***@***.***>
> 

Eric Jett

**Reply by thecodrr** [reply]
Yes that would be great!

**Comment by thecodrr** [comment, marked as answer]
This issue has been resolved in v2.0.4 of the Notesnook Importer!

**Comment by ejjett** [comment]
Thanks!!

On Wed, Aug 30, 2023, at 06:55, Abdullah Atta wrote:
> 
> 
> This issue has been resolved in v2.0.4 of the Notesnook Importer!
> 
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/streetwriters/notesnook/discussions/3215#discussioncomment-6864676>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BCEXSLJBZGUZ34BAVUDCWRDXX5A4LANCNFSM6AAAAAA37A7E6I>.
> You are receiving this because you authored the thread.Message ID: ***@***.***>
> 

Eric Jett

**Comment by ejjett** [comment]
Success, I just imported 969 notes with their corresponding 18 tags. Thank you very much.

On Wed, Aug 30, 2023, at 07:07, Eric Jett wrote:
> Thanks!!
> 
> On Wed, Aug 30, 2023, at 06:55, Abdullah Atta wrote:
>> 
>> 
>> This issue has been resolved in v2.0.4 of the Notesnook Importer!
>> 
>> 
>> 
>> —
>> Reply to this email directly, view it on GitHub <https://github.com/streetwriters/notesnook/discussions/3215#discussioncomment-6864676>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BCEXSLJBZGUZ34BAVUDCWRDXX5A4LANCNFSM6AAAAAA37A7E6I>.
>> You are receiving this because you authored the thread.Message ID: ***@***.***>
>> 
>> 
> 
> Eric Jett

Eric Jett

**Comment by ejjett** [comment]
Brother I really appreciate this, I've been a Standard Notes user for a long time and use the app every day. I'm going to give Notes Nook a run for it's money over the next few days but I already like what I see.

Is this the best place to give you feedback?

Thank,
Eric

On Wed, Aug 30, 2023, at 14:47, Eric Jett wrote:
> Success, I just imported 969 notes with their corresponding 18 tags. Thank you very much.
> 
> On Wed, Aug 30, 2023, at 07:07, Eric Jett wrote:
>> Thanks!!
>> 
>> On Wed, Aug 30, 2023, at 06:55, Abdullah Atta wrote:
>>> 
>>> 
>>> This issue has been resolved in v2.0.4 of the Notesnook Importer!
>>> 
>>> 
>>> 
>>> —
>>> Reply to this email directly, view it on GitHub <https://github.com/streetwriters/notesnook/discussions/3215#discussioncomment-6864676>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BCEXSLJBZGUZ34BAVUDCWRDXX5A4LANCNFSM6AAAAAA37A7E6I>.
>>> You are receiving this because you authored the thread.Message ID: ***@***.***>
>>> 
>>> 
>> 
>> Eric Jett
> 
> Eric Jett
> 

Eric Jett
